source/reference/methods.txt

```diff
@@ -356,6 +356,62 @@ Connection Methods
 
      - Description
 
+   * - :method:`Mongo()`
+
+     - JavaScript constructor to instantiate a database connection from
+       the ``mongo`` shell or from a JavaScript file.
+
+       The :method:`Mongo()` method takes a parameter called
+       ``ClientSideFieldLevelEncryptionOptions`` with the following options:
+
+       .. list-table::
+          :widths: 15 20 40
+          :header-rows: 1
+
+          * - Option
+
+            - Type
+
+            - Description
+
+          * - ``keyVaultClient``
+
+            - :method:`Mongo()` connection object
+
+            - *(Optional)* The MongoDB cluster hosting the key vault
+              collection. Omit to use the current database connection
+              as the key vault host.
+
+          * - ``keyVaultNamespace``
+
+            - String
+
+            - *(Required)* The full namespace of the key vault collection.
+
+          * - ``kmsProvider``
```

Comment:
Ouch -- I just noticed that this is actually a typo in the old shell's documentation. This should be `kmsProviders`, not `kmsProvider`. Unfortunately, we relied on the documentation here, so currently mongosh would expect `kmsProvider` -- I'll try to get this fixed within the next hour or so, so we're not releasing the wrong thing: https://jira.mongodb.org/browse/MONGOSH-604

Reply:
Ah sorry about that!